lis/dataassim/obs/ESACCI_sm/read_ESACCIsm.F90

```diff
@@ -159,8 +161,13 @@ subroutine read_ESACCIsm(n,k,  OBS_State, OBS_Pert_State)
      do c=1,LIS_rc%obs_lnc(k)
         if(LIS_obs_domain(n,k)%gindex(c,r).ne.-1) then 
            grid_index = c+(r-1)*LIS_rc%obs_lnc(k)
+           
+           if(ESACCI_sm_struc(n)%midnight_assimilation.eq..true.) then
+              dt = (LIS_rc%gmt)*3600.0 ! assimilate at 0 UTC
+           else
+              dt = (LIS_rc%gmt - ESACCI_sm_struc(n)%smtime(c,r))*3600.0 ! assimilate at local noon
```

Comment:
Just a note: I am not sure about the minus (and whether it shouldn't be a plus), b/c I don't know what exactly is in smtime at this time.

Reply:
Would it make sense here to comment on what smtime is?